TYPO3.Flow/Migrations/Postgresql/Version20151110113651.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace TYPO3\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration,
+    Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your need!
+ */
+class Version20151110113650 extends AbstractMigration
```

Comment:
not entirely sure here, but I think we decided not to reuse the migration version number for MySQL and Postgres.. @kdambekalns can probably confirm or deny